src/utils/hooks/useKubevirtUserSettings/useKubevirtUserSettings.ts

```diff
@@ -25,20 +26,20 @@ const useKubevirtUserSettings: UseKubevirtUserSettings = (key) => {
   const [error, setError] = useState<Error>();
   const [userName, setUserName] = useState<string>();
   const [userSettings, setUserSettings] = useState<UserSettingsState>();
-  const [loading, setLoading] = useState<boolean>(false);
+  const [loading, setLoading] = useState<boolean>(true);
```

Comment:
is this loading meant for the userSettings? if yes, maybe rename it to clarify what is this loading for.